lib/notifications/client.rb

```diff
@@ -45,7 +45,7 @@ def send_sms(args)
     # @return [Notification]
     def get_notification(id)
       Notification.new(
-        speaker.send(:get, id)["notification"]
+        speaker.send(:get, id)["data"]["notification"]
       )
     end
```

Comment:
This interface looks a bit brittle because this code has specific knowledge of the data structure, i.e. `["data"]["notifications"]`. Maybe that's fine. You could be defensive and use [fetch](http://ruby-doc.org/core-2.2.0/Hash.html#method-i-fetch) instead, which will raise a KeyNotFound error if your assumption is not correct and the data structure does not contain those keys. Might be worth considering.